_posts/2020-08-17-creating-your-ci-cd-pipeline-with-concourse.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: post
+title: Creating your CI/CD-pipeline with Concourse
+author: alperttiti
+excerpt: >
+  Choosing a CI/CD-system is very much the same thing as choosing a good server name.
+  You probably have a long ride with your decision ahead. Concourse CI is one option.
+tags:
+ - Concourse CI
+ - Build pipeline
+ - Continuous Integration
+ - Continuous Delivery
+ - DevOps
+---
+
+There are many CI/CD systems available. If you are using Github you have probably already
+tried Github Actions which are a great way to build and deploy your *things*, even it's
+still in its early days. If you work mainly in some cloud environment you probably have
+tried tools available on that specific environment. There is also *good old Jenkins* and many more.
+
+However, this blog post is about Concourse CI. Concourse is an open-source continuous
+*thing-doer* like they have put it on their [website](https://concourse-ci.org/).
+The most common way to run a Concourse binary is to use [Docker](https://www.docker.com/), so
+let's go on that path. I will show you how to create a simple pipeline with Concourse.
+
+<img style="width: 90%; display: block; margin-left: auto; margin-right: auto;" src="/img/creating-your-ci-cd-pipeline-with-concourse/concourse_example_pipeline.png" alt="Concourse example pipeline" />
+
+Before we'll get further, it's a good time for a disclaimer. Before you just start *hacking*,
+it's good to read a little what Concourse is and what direction it's heading in the future. A good place to start is
+their [blog post about the roadmap towards v10](https://blog.concourse-ci.org/core-roadmap-towards-v10/).
+Core developers behind Concourse have also said, that Concourse is not for everyone and for every need.
+For example, [they have ruled parametrized jobs out of their scope.](https://github.com/concourse/concourse/issues/783)
+Also, Concourse is nothing new and fancy, it has been there since the year 2014.
+
+Anyway, because you still be reading this, you probably think as I do, 
```

Comment:
Anyway, as you are still reading this you might feel the same as I do - one cannot know if it's good or not without trying it.